activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+*   Optimize Active Record batching further when using ranges.
+
+    When used for full table iterations, or with the `use_ranges: true` option, `in_batches`
```

Comment:
While this info is inspiring and interesting, it is pretty long and I am not sure that users reading the CHANGELOG need to read this. Maybe paste these into the commit body itself?

For this change, I would leave only the first line and the last one (starting with `E.g., tested ....`)

Reply:
Good point, I made that changelog entry way too complicated.
I've updated it as you suggested.